### PR TITLE
[FEATURE] Skip metadata sync for AlluxioRuntime

### PR DESCRIPTION
**Pull request by TrafalgarZZZ**
### Ⅰ. Describe what this PR does
Support skipping metadata sync for AlluxioRuntime. Via setting `AlluxioRuntime.spec.skipMetadataSync` to `true`, users can explicitly disable metadata loading process when setting up an Alluxio instance. For example: 

```
kind: AlluxioRuntime
...
spec:
  skipMetadataSync: true
```

This option prevents Fluid controller from loading all the metadata from UFS and avoid laying high pressure on Alluxio's master when there is a large amount of files and paths in the UFS.

However, NOTE THAT this option:
- stops showing dataset summary information (e.g. UFS Total Size & Cached Percentage)
- may negatively affect the efficiency when querying a specific file or path at the first time. Get more information about Alluxio's properties `alluxio.user.file.metadata.sync.interval` and `alluxio.user.file.metadata.load.type` [here](https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html)

 

### Ⅱ. Does this pull request fix one issue?

#2577 
#2133 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: api/v1alpha1/alluxioruntime_types.go [hunk 1]
```diff
@@ -295,6 +295,10 @@ type AlluxioRuntimeSpec struct {
 	// CleanCachePolicy defines cleanCache Policy
 	// +optional
 	CleanCachePolicy CleanCachePolicy `json:"cleanCachePolicy,omitempty"`
+
+	// SkipMetadataSync defines whether to skip syncing metadata when Alluxio is being set up. Default to false.
+	// +optional
+	SkipMetadataSync bool `json:"skipMetadataSync,omitempty"`
```
Comment (cheyang) [inline api/v1alpha1/alluxioruntime_types.go:301]: How about `disableMetadataSync`?

Comment (TrafalgarZZZ) [reply api/v1alpha1/alluxioruntime_types.go]: Made `metadataSyncBehavior` and `cacheCleanBehavior` sub-properties of `AlluxioRuntime.spec.management`. We shall define interactions between runtime engine and operator in `AlluxioRuntime.spec.management` in future development

File: api/v1alpha1/eacruntime_types.go [hunk 2]
```diff
@@ -154,7 +154,7 @@ type EACRuntimeSpec struct {
 
 	// CleanCachePolicy defines cleanCache Policy
 	// +optional
-	CleanCachePolicy CleanCachePolicy `json:"cleanCachePolicy,omitempty"`
+	CleanCachePolicy CleanCacheBehavior `json:"cleanCachePolicy,omitempty"`
```
Comment (cheyang) [inline api/v1alpha1/eacruntime_types.go:157]: I've checked some other projects. Seems policy is widely used https://github.com/kubeflow/common/blob/master/pkg/apis/common/v1/types.go#L146 than behavior.

**Review by cheyang — APPROVED** [review]

**Pull request merged**